### PR TITLE
[samples/java]: Add some more descriptive text to the README

### PR DESCRIPTION
**Pull request by asimshankar**
CC @random-forests

### CODE REVIEW

File: samples/languages/java/README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 # TensorFlow for Java: Examples
 
-Examples using the TensorFlow Java API.
+These examples include using pre-trained models for [image
+classification](label_image) and [object detection](object_detection),
+and driving the [training](training) of a pre-defined model - all using the
+TensorFlow Java APIs.
```
Comment (wolffg) [inline samples/languages/java/README.md]: Nit: Is it plural or not?  I think it's not.

File: samples/languages/java/README.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+
+The TensorFlow Java API does not have feature parity with the Python API.
+The Java APIs are most suitable for inference using pre-trained models
+and for training pre-defined models from a single Java process.
+
+However, Python will be the most convenient language for defining the
```
Comment (wolffg) [inline samples/languages/java/README.md]: Nit: I think you can remove "However".

File: samples/languages/java/README.md (hunk 1 above)
Comment (asimshankar) [reply samples/languages/java/README.md]: Made singular.

File: samples/languages/java/README.md (hunk 2 above)
Comment (asimshankar) [reply samples/languages/java/README.md]: Done.

**Comment by asimshankar** [comment]
Thanks!

**Comment by nealwu** [comment]
@MarkDaoust do we need this commit in the r1.5 branch?

**Comment by asimshankar** [comment]
@nealwu : Nope. This isn't referenced from the website at this point.

**Comment by nealwu** [comment]
Ok sounds good, just making sure we double check.

**Pull request merged**